src/io/appium/settings/LocationService.java

```diff
@@ -0,0 +1,28 @@
+package io.appium.settings;
+
+import android.app.Service;
+import android.content.Intent;
+import android.location.LocationManager;
+import android.os.IBinder;
+import android.util.Log;
+
+public class LocationService extends Service {
+
+    MockLocationProvider mock;
+
+    @Override
+    public IBinder onBind(Intent intent) {
+        return null;
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId) {
+        Log.i("LocalService", "setting the location from service");
```

Comment:
The log should what location is being set (long/lat), not just that we're setting a location.